ee-feature-pack/common/src/main/resources/modules/system/layers/base/io/opentelemetry/trace/main/module.xml

```diff
@@ -0,0 +1,51 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<module xmlns="urn:jboss:module:1.9" name="io.opentelemetry.trace">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+    <resources>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk-trace}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk-common}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-semconv}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-api-metrics}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-exporter-jaeger}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-exporter-otlp-common}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-exporter-otlp-trace}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-proto}"/>
+    </resources>
+
+    <dependencies>
+        <module name="io.opentelemetry.api"/>
+        <module name="io.opentelemetry.context"/>
+        <module name="com.squareup.okhttp3"/>
+        <module name="org.slf4j"/>
+        <module name="io.jaegertracing"/>
+        <module name="io.grpc"/>
+        <module name="com.google.guava" />
+        <module name="com.google.code.gson"/>
+
+        <module name="java.logging"/>
+        <module name="sun.jdk"/>
```

Comment:
sun.jdk is deprecated. In [https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/common/src/main/resources/modules/system/layers/base/sun/jdk/main/module.xml](https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/common/src/main/resources/modules/system/layers/base/sun/jdk/main/module.xml) you can find the modules to use instead.

Reply:
Fixed.